apps/src/sites/studio/pages/congrats/index.js

```diff
@@ -47,21 +44,21 @@ $(document).ready(function () {
       isHocTutorial,
       isPlCourse,
       isK5PlCourse,
+      courseNames: [courseName],
```

Comment:
I made this plural as we'll pass down multiple course names when multiple certificates come in